src/components/aeIdentityMain/aeIdentityMain.scss

```diff
@@ -0,0 +1,125 @@
+@import './../variables';
+
+.ae-identity {
+  width: 100%;
+  zoom: 1.1;
+  color: $white;
+}
+
+.flex-row {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.chunk-row {
+  font-family: 'Roboto Mono', monospace;
+  font-size: 17px;
+  font-weight: 300;
+  font-style: normal;
+  line-height: 1.53;
+  letter-spacing: 3.1px;
+  font-stretch: normal;
+  text-align: left;
+
+  &:first-of-type {
+    margin-top: 14px;
+  }
+}
+
+.chunk {
+  display: inline-block;
+  width: calc(100% / 3);
+
+  &:nth-child(2n) {
+    text-align: center;
+  }
+
+  &:nth-child(3n) {
+    text-align: right;
+  }
+}
+
+.flex-row + .chunk-row {
+  margin-top: 16px;
+}
+
+.identity-info._long {
+  width: calc(100% - 65px);
+  overflow: hidden;
+}
+
+.balances {
+  width: auto;
+  flex-grow: 1;
+}
+
+.balance {
+  font-family: 'Roboto Mono', monospace;
+  font-size: 12px;
+  font-weight: 300;
+  font-stretch: normal;
+  line-height: 12px;
+  text-align: right;
+  margin-top: 2px;
+
+  &:first-of-type {
+    margin-top: 0;
+  }
+
+  &._bold {
+    font-weight: 400;
+  }
+}
+
+.currency-symbol {
+  letter-spacing: 0.12em;
+  padding-right: 3px;
+}
+
+.ae-identity__avatar {
+  border-width: 1px !important;
+  width: 23px !important;
+  height: 23px !important;
+}
+
+.identity-name-position {
+  &._collapsed {
+    &_true {
+      margin-left: 9px;
+    }
+    &_false {
+      margin-left: 10px;
+    }
+  }
+}
```

Comment:
`_collapsed_true/false` makes code too verbose, why don't define just `_collapsed` and override default style with it?
```
.identity-name-position {
  margin-left: 10px;
  &._collapsed {
    margin-left: 9px;
  }
}
```